Assets/MRTK/Providers/Windows/WindowsSpeechInputProvider.cs

```diff
@@ -197,11 +202,14 @@ public override void Update()
 
                 if (keywordRecognizer != null && keywordRecognizer.IsRunning)
                 {
-                    for (int i = 0; i < Commands.Length; i++)
+                    SpeechCommands[] commands = Commands;
```

Comment:
Not sure I understand why we create a local copy of the commands here.

Reply:
Same reason as https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10601#discussion_r878489717